SitumWayfinding/Classes/Positioning/Internal/LocationManagerFactory.swift

```diff
@@ -84,6 +84,8 @@ fileprivate class FakeLocationManager: NSObject, SITLocationInterface {
     
     func updateLocationParameters(_ update: SITLocationParametersUpdate) {}
     
+    func add(_ externalLocation: SITExternalLocation) {}
```

Comment:
En Android este método chámase **addExternalLocation()**, queremos manter cohesión entre plataformas ?

Reply:
Ademáis, onde está a lóxica de procesado das SITExternalLocation ? Porque non existía este método público cando se fixo a implementación ?

Reply:
Ao ser .swift non se necesita declarar este método público nunha interfaz ?